packages/form-builder/src/blocks/fields/amount/index.js

```diff
@@ -0,0 +1,37 @@
+import {Icon} from '@wordpress/icons';
+import {__} from "@wordpress/i18n";
+import settings from "../settings";
+
+import Edit from './edit';
+
+const amount = {
+
+    name: 'custom-block-editor/donation-amount-levels', // @todo Rename this block.
+
```

Comment:
Has this todo been completed?

Reply:
I left the todo because I hadn't decided on a new name yet, but I'll being doing a pass soon to rename all the block namespaces and will do it then.